lib/blockscore/actions/update.rb

```diff
@@ -0,0 +1,56 @@
+# Public: Contains the :save instance method, which updates the object with
+# the BlockScore API to persist the changes.
+#
+# Examples
+#
+#  class Foo
+#    include BlockScore::Actions::Update
+#  end
+#
+#  foo = Foo.new
+#  foo.name_first = 'John'
+#  foo.save
+#  # => true
+module BlockScore
+  module Actions
+    module Update
+      # Public: Saves the changes to the object via an Update call to
+      # BlockScore API.
+      #
+      # Returns true if the update is successful, false otherwise.
+      def save
+        self.class.patch "#{self.class.endpoint}#{id}", filter_params
+        true
+      rescue
```

Comment:
Be specific with `rescue` blocks. This will catch ANY exception raised.  It also looks like HTTParty doesn't raise exceptions unless something is actually very wrong too, so what was the intention?